### PR TITLE
chore: restored vue.set

### PR DESCRIPTION
**Pull request by molimauro**
### What this PR does 📖
- Title

### Which issue(s) this PR fixes 🔨
- Resolve #

### CODE REVIEW

File: libraries/Iridium/webrtc/WebRTCManager.ts [hunk 1]
```diff
@@ -314,18 +315,13 @@ export default class WebRTCManager extends Emitter {
...
-      [did]: {
-        audio: audio ?? this.state.streamMuted[did]?.audio ?? false,
-        video: video ?? this.state.streamMuted[did]?.video ?? false,
-        screen: screen ?? this.state.streamMuted[did]?.screen ?? false,
-        headphones:
-          headphones ?? this.state.streamMuted[did]?.headphones ?? false,
-      },
-    }
-
-    this.emit('streamMuted', { did, ...this.state.streamMuted[did] })
```
Comment (josephmcg) [inline libraries/Iridium/webrtc/WebRTCManager.ts:328]: not sure what this does, but just checking it wasnt accidentally removed

File: libraries/Iridium/friends/FriendsManager.ts [hunk 2]
```diff
@@ -447,14 +438,9 @@ export default class FriendsManager extends Emitter<IridiumFriendPubsub> {
...
     request.status = 'accepted'
-    delete this.state.requests[did]
     this.state.friends = [...this.state.friends, did]
-    this.state.requests = Object.keys(this.state.requests)
-      .filter((key) => key !== did)
-      .reduce((acc, key: string) => {
-        acc[key] = this.state.requests[key]
-        return acc
-      }, {} as { [key: string]: FriendRequest })
+    Vue.delete(this.state.requests, did)
```
Comment (josephmcg) [inline libraries/Iridium/friends/FriendsManager.ts:442]: there's a spread here on 441, not entirely sure what it's doing. maybe a push would be just as good?

**Review by josephmcg — COMMENTED** [review]
two minor things, other than that looks good

File: libraries/Iridium/webrtc/WebRTCManager.ts (hunk 1 above)
Comment (molimauro) [reply libraries/Iridium/webrtc/WebRTCManager.ts:328]: there's no `.on` listener for that around the app

File: libraries/Iridium/friends/FriendsManager.ts (hunk 2 above)
Comment (molimauro) [reply libraries/Iridium/friends/FriendsManager.ts:442]: it has been always like that in the past

Comment (josephmcg) [reply libraries/Iridium/friends/FriendsManager.ts:442]: gotcha, just a stylistic thing then. this definitely works.

**Review by josephmcg — APPROVED** [review]

**Pull request merged**